tools/gfx/render.h

```diff
@@ -938,23 +1013,23 @@ class Renderer: public Slang::RefObject
     }
 
     virtual Result createGraphicsPipelineState(
-        const GraphicsPipelineStateDesc&    desc,
+        GraphicsPipelineStateDesc&    desc,
```

Comment:
Why are the `const`s getting removed here? That seems incorrect. Creating an object shouldn't modify the `desc` that was used to create it, right?